documentation/asciidoc/topics/proc_connecting_cli.adoc

```diff
@@ -6,15 +6,22 @@ Use the command line interface (CLI) to connect to your {brandname} cluster and
 
 .Prerequisites
 
-* Download the server distribution so you can run the CLI.
+* Download a CLI distribution so you can connect to {brandname} clusters on {openshiftshort}.
 
-The CLI is available as part of the server distribution, which you can run on your local host to establish remote connections to {brandname} clusters on {openshiftshort}.
-
-//Community
+//Community content
 ifdef::community[]
-Alternatively, you can use the infinispan/cli image at link:https://github.com/infinispan/infinispan-images[https://github.com/infinispan/infinispan-images].
+The CLI is available as part of the server distribution.
+Alternatively, you can use the CLI container image available from link:https://infinispan.org/download/stable/cli-image.html[infinispan.org/download].
 endif::community[]
 
+//Downstream content
+ifdef::downstream[]
+The {brandname} CLI is available with the server distribution or as a native executable.
+
+Follow the instructions in _Getting Started with {brandname} Server_ for information on downloading and installing the CLI as part of the server distribution.
+For the native CLI, you should follow the installation instructions in the _README_ file that is included with the distribution.
```

Comment:
what README file?

Reply:
README.adoc - that's in the zip that comes with CR2

Reply:
I'm not sure if it is clear. the word "distribution" is always after "server" and users may think the native executable is in the "server distribution"

Reply:
gotcha. I'll reword. thanks!